tests/drum/drum_server_utils.py

```diff
@@ -62,22 +62,27 @@ def __init__(
         with_error_server=False,
         show_stacktrace=True,
         nginx=False,
+        memory=None,
+        fail_on_shutdown_error=True,
```

Comment:
Shouldn't   `fail_on_shutdown_error` var be called smth like `shutdown_on_fail` ?

Reply:
The idea here is not to fail on a shutdown error.. 
In the   test I added I am actually killing the container while inside the the scope of the context manager. The issue is that when going out of the scope of the context manager we go into the __exit__ function and then we try to send a shutdown request to the server in the docker (in case of nginx=None). Since the docker is down (we just killed it by asking r2d2 to allocate 1000mb where --memory was 500).. the request is throwing an exception..